src/lib/services/state-service.ts

```diff
@@ -266,6 +290,7 @@ export default class StateService {
                         this.featureEnvironmentStore.connectFeatureToEnvironmentsForProject(
                             env.featureName,
                             id,
+                            featureEnvironments,
```

Comment:
Send configurations to set the desired "enabled state" of the feature in each environment (note that `connectFeatureToEnvironmentsForProject` will iterate and create connections for all environments)